### PR TITLE
Add blog post for release 5.4

### PR DESCRIPTION
**Pull request by keertk**
Tracking issue: https://github.com/bazelbuild/bazel/issues/16663
Expected publish date: 2022-12-13

### CODE REVIEW

**Review by aiuto — COMMENTED** [review]
LGTM

**Comment by Wyverald** [comment]
@aiuto, we need an actual "approval" on the GitHub UI, just the string "LGTM" unfortunately isn't enough :P

**Comment by aiuto** [comment]
I can do the approve, but I thought it should be a PM that does that actual approve, so Radika, instead of me.

**Review by aiuto — APPROVED** [review]

**Comment by aiuto** [comment]
We should probably fix the CODEOWNERS for this repo.  hicksjoseph should be out. @meteorcloudy should be in.

**Comment by meteorcloudy** [comment]
Created https://github.com/bazelbuild/bazel-blog/pull/287

**Comment by keertk** [comment]
Closing - no blog post for minor releases moving forward.

**Pull request closed without merging**
